src/CBT.Core/CBT.Core.csproj

```diff
@@ -5,6 +5,7 @@
   <ItemGroup>
     <Reference Include="Microsoft.Build, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
     <Reference Include="Microsoft.Build.Framework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
+    <Reference Include="Microsoft.Build.Utilities.v4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
```

Comment:
I would really like to avoid referencing this assembly since it won't be redirected by MSBuild or Visual Studio.  I think you're only using it to implement `Task`, if so, just implement `ITask` instead?